ecr_api.py

```diff
@@ -190,25 +190,28 @@ def __init__ ( self , retries=60) :
     # returns true if user has any of the permissions 
     def hasPermission(self, app_id, granteeType, grantee, permissions):
 
+        
+        permissionOR = "permission = %s" + " OR permission = %s" * (len(permissions) -1)
```

Comment:
Are you able to just use an IN query instead? That's subjectively clearer to me.

```sql
permission in ('perm1', 'perm2', ...)
```

Reply:
I replaced the OR with an IN as you suggested. But since the number of permissions is variable it is still ugly. Or did you have something else in mind ?

Reply:
I was just curious if Python allowed you to use a single template for a tuple or not instead of having to manually add n explicit %s in the query. It's fine for now, though.